Microsoft.Toolkit.Uwp.UI.Controls.Primitives/VisualStudioToolsManifest.xml

```diff
@@ -0,0 +1,11 @@
+<FileList>
+  <File Reference="Microsoft.Toolkit.Uwp.UI.Controls.Primitives.dll">
+    <ToolboxItems VSCategory="Windows Community Toolkit" BlendCategory="Windows Community Toolkit">
+      <Item Type="Microsoft.Toolkit.Uwp.UI.Controls.AdaptiveGridView" />
+      <Item Type="Microsoft.Toolkit.Uwp.UI.Controls.DockPanel" />
+      <Item Type="Microsoft.Toolkit.Uwp.UI.Controls.StaggeredPanel" />
+      <Item Type="Microsoft.Toolkit.Uwp.UI.Controls.UniformGrid" />
+      <Item Type="Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel" />
```

Comment:
If you don't mind can you add SwitchPresenter here? We probably need a small guide on what we need to add to the designer projects for new controls, as I don't think we audit this currently in our PRs, probably a step we should add to our template list?

Reply:
4f83e08